generated-dockerfiles/7.4/fpm-nginx/etc/nginx/server-opts.d/performance.conf

```diff
@@ -14,6 +14,8 @@ location = /robots.txt {
 location ~* \.(?:css(\.map)?|js(\.map)?|jpe?g|png|gif|ico|cur|heic|webp|tiff?|mp3|m4a|aac|ogg|midi?|wav|mp4|mov|webm|mpe?g|avi|ogv|flv|wmv)$ {
     expires    7d;
     access_log off;
+    # Pass to PHP to help support things like Laravel Livewire
```

Comment:
I propose changing the comment to

`# Pass to PHP to ensure PHP apps can handle routes that end in these filetypes`

as this is neither Livewire, nor Laravel specific. The same would happen if you server a route ending in .css with e.g. Yii.

Reply:
Removing those comments is better, thought.

Reply:
I love having the comment in there because it reminds me on why it's there 😆

I also added this:
```
access_log off;
log_not_found off;
```

Thanks guys!